nbbuild/templates/common.xml

```diff
@@ -532,6 +532,11 @@
                 <isset property="test-have-jdk-1.9"/>
             </or>
         </condition>
+        <condition property="test.tracking.agent.dir" value="${nbplatform.active.dir}/platform/core">
+            <available file="${nbplatform.active.dir}/platform/core" type="dir" />
+        </condition>
+        <property name="test.tracking.agent.dir" location="${o.n.bootstrap.dir}/core/" />
+        <property name="test.tracking.agent.args" value="-javaagent:${test.tracking.agent.dir}/tracking-agent.jar -Xbootclasspath/a:${test.tracking.agent.dir}/tracking-hooks.jar"/>
```

Comment:
Having to manually modify `-Xbootclasspath` is annoying. Can't agent modify classpath programatically? There are methods like [appendToBootstrap...](https://docs.oracle.com/javase/8/docs/api/java/lang/instrument/Instrumentation.html#appendToBootstrapClassLoaderSearch-java.util.jar.JarFile-).